daily_word_bot/db.py

```diff
@@ -62,6 +65,27 @@ def remove_user_blocked_word(self, message, word_id):
     def get_user_blocked_words(self, chat_id) -> typing.List[str]:
         return to_string_list(self.r.smembers(f"blockedWords-{chat_id}"))
 
+    def get_user_levels(self, chat_id) -> typing.List[str]:
+        user_info = self.get_user(chat_id)
+        user_levels = user_info["levels"] if "levels" in user_info else []
+        return user_levels
+
+    def remove_user_level(self, chat_id, level) -> None:
+        user_info = self.get_user(chat_id)
+        levels = user_info["levels"]
```

Comment:
true! But I thought that implementing this feature in production would mean an update of the users making each one have the column with a default value. What do you think about that?

Reply:
My suggestion would be using `user_info.get("levels")` since this does not raise an Exception when the key is missing.

we could do `user_info.get("levels", [])` to fallback to empty list